hammerblade/torch/tests/test_hammerblade_div.py

```diff
@@ -0,0 +1,50 @@
+"""
+Tensor division tests
+
+Bandhav Veluri
+"""
+
+import torch
+
+def test_elementwise_div_1():
+    x = torch.ones(1, 10)
+    y = torch.ones(1, 10)
```

Comment:
Testing dividing two `ones` tensor seems not that useful though ...

Reply:
Dividing ones is more of an indexing test for division kernel than a test for the actual division.